client/rest-high-level/src/test/java/org/elasticsearch/client/AbstractHlrcStreamableXContentTestCase.java

```diff
@@ -14,6 +27,8 @@
 
 import static org.elasticsearch.test.AbstractXContentTestCase.xContentTester;
 
+// TODO: Remove and change subclasses to use AbstractResponseTestCase instead
+@Deprecated
```

Comment:
Can you add a comment w/ @deprecated please use this.class.java instead for our IDE friends.

Reply:
pushed: f9f217e911b9971b9e70066d5e70510928a0caa6